### DISCUSSION TITLE
Failed to load next.config.js

### DISCUSSION THREAD

**Discussion by devwilliamy** [muxinc/next-video#153] [Q&A]
Hi,

This is probably a duplicate of #117 

I'm on Next 14.1.0 and Next 14.0.4 (tried making a new project and also trying to integrate into my own)

I've run `npx next-video init` and it's able to detect that I have a `next.config.js`

This is my `next.config.js` and it's in my root directory:

```
const { withNextVideo } = require('next-video/process');
/** @type {import('next').NextConfig} */
const nextConfig = {};

module.exports = withNextVideo(nextConfig);
```

Oddly enough, it does seem to upload to Mux, but I'm just guessing it's because Mux is the default.

Should I worry about not being able to load `next.config.js`?

**Comment by luwes** [comment]
yes, the script should be able to load next.config.js when running `npx next-video sync` in the root of your project once you added the required env variable for Vercel blob `BLOB_READ_WRITE_TOKEN`

here is an online example with Vercel blob
https://codesandbox.io/p/devbox/next-video-vercel-blob-w6wwqw?file=%2Fapp%2Fpage.js%3A14%2C17

can you get that to work?

**Reply by devwilliamy** [reply]
Hi @luwes , I think we can close this one and continue this discussion on #151 

TL;DR I added `BLOB_READ_WRITE_TOKEN` but it would still say failed to load next.config.js

**Comment by luwes** [comment]
fixed in https://github.com/muxinc/next-video/pull/159
